File: 2024/FASE 5 - OOP/PrimeiroProjeto/PrimeiroProjeto/src/Calculadora.java
import java.util.Scanner;

public class Calculadora {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);

        System.out.print("Digite o primeiro valor: ");
        double primeiroValor = scanner.nextDouble();
        System.out.print("Digite o segundo valor: ");
        double segundoValor = scanner.nextDouble();

        double soma = primeiroValor + segundoValor;
        double subtracao = primeiroValor - segundoValor;
        double multiplicacao = primeiroValor * segundoValor;
        double divisao = primeiroValor / segundoValor;

        System.out.println("Soma: " + soma);
        System.out.println("Subtração: " + subtracao);
        System.out.println("Multiplicação: " + multiplicacao);
        System.out.println("Divisão: " + divisao);

        scanner.close();
    }
}

File: 2024/FASE 1 - DEVELOPMENT ENVIRONMENT/Cap 10 - Python/Calculadora.py
valor1 = input("Digite o primeiro valor:");
valor2 = input("Digite o segundo valor:");
soma = float(valor1) + float(valor2);
print("A soma de", valor1, "+", valor2, "é igual a {}".format(soma));
subtracao = float(valor1) - float(valor2);
print("A subtracao de", valor1, "-", valor2, "é igual a {}".format(subtracao));
divisao = float(valor1) / float(valor2);
print("A divisão de", valor1, "/", valor2, "é igual a {}".format(divisao));
multiplicacao = float(valor1) * float(valor2);
print("A multiplicação de", valor1, "*", valor2, "é igual a {}".format(multiplicacao));

#Tipos de conversões
# Tipos básicos  |          Descrição          | Exemplo
#     int               Números inteiros        1, 2, 100
#    float       Números reais(ponto flutuante) 1.5, 2.07, 50.29
#   complex             Números complexos       4j, 5+2j, 15j
#    bool                Valores lógicos        True, False, 1, 0
#   string                   Textos             "Texto", "a", "10"